src/tests/ftest/server/daos_server_dump.py

```diff
@@ -25,43 +26,99 @@ def __init__(self, *args, **kwargs):
         self.start_servers_once = False
         self.setup_start_agents = False
 
+    def tearDown(self):
+        """Tear down after each test case."""
+        super().tearDown()
+
+        # force test status !!
+        # use mangling trick described at
+        # https://stackoverflow.com/questions/3385317/private-variables-and-methods-in-python
+        # to do so
+        self._Test__status = 'PASS'
+
+        # DAOS-1452 may need to check for one file per engine...
+        ret_codes = pcmd(self.hostlist_servers, r"ls /tmp/daos_dump*.txt")
+        # Report any failures
+        if len(ret_codes) > 1 or 0 not in ret_codes:
+            failed = [
+                "{}: rc={}".format(val, key)
+                for key, val in ret_codes.items() if key != 0
+            ]
+            print(
+                "no ULT stacks dump found on following hosts: {}".format(
+                ", ".join(failed)))
+            self._Test__status = 'FAIL'
+
     def test_daos_server_dump_basic(self):
         """JIRA ID: DAOS-1452.
 
-        Test Description: Test engine ULT stacks dump.
+        Test Description: Test engine ULT stacks dump (basic).
 
-        :avocado: tags=all,daily_regression
-        :avocado: tags=vm
-        :avocado: tags=control,server_start,basic
-        :avocado: tags=daos_server_dump_test,test_daos_server_dump_basic
+        :avocado: tags=all
+        :avocado: tags=daos_server_dump_tests,test_daos_server_dump_basic
         """
 
-        ret_codes = stop_processes(self.hostlist_servers, r"daos_engine",
-                           added_filter=r"'\<(grep|defunct)\>'",
-                           dump_ult_stacks=True)
+        ret_codes = dump_engines_stacks(self.hostlist_servers,
+                           added_filter=r"'\<(grep|defunct)\>'")
+        # at this time there is no way to know when Argobots ULTs stacks
+        # has completed, see DAOS-1452/DAOS-9942.
         if 1 in ret_codes:
             print(
```

Comment:
This should be a `self.log.info()`

Reply:
As I mentioned in my main review comment this doesn't have to be addressed in this cherry-pick, but we should clean it up in master at some point.